at/tensor.go

```diff
@@ -0,0 +1,54 @@
+package at
```

Comment:
The meaning of `at` is not clear. This package is actually a go wrapper of [aten](https://github.com/pytorch/pytorch/tree/master/aten) library of PyTorch. How about naming it to `aten`?

Reply:
The RandN is not the one from the C++ namespace ten, but the one from the namespace torch.  If we are going to move, we can name the new package torch.

Reply:
Good idea, I separated the `at` package into `aten` and `torch` packages. 
As the documentation https://golang.org/cmd/cgo/ says:
> Cgo translates C types into equivalent unexported Go types. Because the translations are unexported, a Go package should not expose C types in its exported API: a C type used in one Go package is different from the same C type used in another.

I added a function `NewTensor(unsafe.Pointer)`, that we can initialize the Tensor which comes from `torch::RandN`.